### ISSUE TITLE
Elementor 3.12.0 Colour Contrast Issues

### ISSUE THREAD

**Issue by nickdgreen** [elementor/elementor#21839]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I don't love the new colour scheme but I don't hate it either - it is what it is. There are, however, several issues with colour contrast with the latest version 3.12.0. in Dark Mode.
1. Text widget text editor - when hovering over items above the text window, for example, a bulleted list the highlighted item is just a white square with a tooltip underneath. (text editor hover.png)
2. Text widget text editor - when text is highlighted the background highlight is a dark purple with white text that provides poor contrast and it is hard to identify what has been highlighted. (perple highlight.png)
3. For any setting that has different options (px,%,EM,REM etc.) the highlighted pink is very hard to see so it is unclear which parameter has been selected. (purple hightlight2.png)
4. For settings with multiple values, for example, padding. It is virtually impossible to see that a value has been selected for adjustment. In the attached image the bottom value has been selected but it's very hard to see that.  (purple hightlight3.png)
5. For settings that have a dropdown, for example, image set sizes the dropdown "Thumbnail 150x150 etc" is pale grey text on a white background. On first encountering this I thought these values were "greyed out" and unavailable. The dropdown needs to have better contrast and definitely not look "greyed out". This was reported in the previous version and I thought it had been fixed. (dropdown contrast.png)

![dropdown contrast](https://user-images.githubusercontent.com/22448787/228809391-584f881f-01b3-4bed-b4f2-4e72c22a8b9b.png)
![text editor hover](https://user-images.githubusercontent.com/22448787/228809393-97252603-aae2-4c15-bd65-9193c7276ff7.png)
![purple highlight 3](https://user-images.githubusercontent.com/22448787/228809398-691e7a6a-83bf-4bf9-a9a9-f38d6749da7f.png)
![purple highlight 2](https://user-images.githubusercontent.com/22448787/228809399-6616d475-508f-45c7-b0ca-4a5d3ab3c041.png)
![perple highlight](https://user-images.githubusercontent.com/22448787/228809401-11e08d39-f055-45ce-b764-14760230ed2c.png)

### Steps to reproduce

Just Darkmode interface on the Elementor Panel.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

Elementor 3.12.0 latest update

**Comment by black-eye** [comment]
+1

**Comment by nicholaszein** [comment]
Hello @nickdgreen! 

Thank you for submitting your issue! 🙏 

🏷️ This issue has been labeled and will be forwarded to the team for validation and possible solution. Please keep in mind that we can't provide any ETA on when this issue could be fixed if it is validated. We ask for patience.

Best regards

**Comment by IMTanuki** [comment]
Just an editorial comment - it's incomprehensible that Elementor would release an update that presumes to make its product more accessible for color-blind users, while making the product nearly unusable for everyone else. This was a major change and should have been vetted with the user community before going GA.

**Comment by nicholaszein** [comment]
> Just an editorial comment - it's incomprehensible that Elementor would release an update that presumes to make its product more accessible for color-blind users, while making the product nearly unusable for everyone else. This was a major change and should have been vetted with the user community before going GA.

Hi @IMTanuki, we had a public beta testing program that run for 3 weeks, and we had lots of feedback.

**Comment by IMTanuki** [comment]
> > Just an editorial comment - it's incomprehensible that Elementor would release an update that presumes to make its product more accessible for color-blind users, while making the product nearly unusable for everyone else. This was a major change and should have been vetted with the user community before going GA.
> 
> Hi @IMTanuki, we had a public beta testing program that run for 3 weeks, and we had lots of feedback.

I don't know who you employed in your beta - or maybe you just ignored the feedback. 3.12 is nearly unusable and the feedback on Reddit and other platforms is consistent that this is a major step backwards.

**Comment by lensbreak** [comment]
> Just an editorial comment - it's incomprehensible that Elementor would release an update that presumes to make its product more accessible for color-blind users, while making the product nearly unusable for everyone else. This was a major change and should have been vetted with the user community before going GA.

EXACTLY! They have never ever read their own blog article about web accessibility xD

**Comment by lensbreak** [comment]
> > Hi @IMTanuki, we had a public beta testing program that run for 3 weeks, and we had lots of feedback.
> 
> Hi @nicholaszein, A good opportunity was found to talk about a topic that I have wanted to talk about for a long time.
> 
> Elementor updates have been very powerful and big for about 3 months. This requires the participation of many, many people to participate in the beta version. Right now, there are maybe only 5 of us who are constantly contributing to beta versions, and that's really low for Elementor, which has millions of users.
> 
> **I think the reason for low user participation is that they don't know how to contribute to beta versions.** On the other hand, many users may not have licensed the pro version to try it out. I think it's very, very important to get as many people involved in the beta version. This makes the official update much more stable(Of course, this requires that on the other hand the bug fixing team can cover the possible bugs reported in the beta version. **The number of team members.**).
> 
> **So I wanted to suggest 3 ways that can definitely increase the number of beta contributors.**
> 
> 1- Whenever a beta version is released, announce it on Instagram, YouTube, and other Elementor platforms in addition to the current email.
> 
> 2- Publish (in full) a beta participation guide on YouTube, Instagram, and even this GitHub. Things like installing the beta version, where to report bugs on GitHub and other things needed.
> 
> 3- I don't know how, but somehow make the pro beta available to users for free. As I said above, there are many people who buy the pro version just for their clients and don't have the pro version for themselves to be able to participate in the pro beta version.
> 
> I hope it was useful.
> 
> thank you

According to Elementor website they have 400 employees and obviously a huge money making business model. If Elementor does not learn from other companies making the mistake "we are the best and nothing can happen to us" they will eventually come to a fall. I hope the managament of Elementor can make a turn and finally focus on the communities wishes.

**Comment by IMTanuki** [comment]
> > > Hi @IMTanuki, we had a public beta testing program that run for 3 weeks, and we had lots of feedback.
> > 
> > 
> > Hi @nicholaszein, A good opportunity was found to talk about a topic that I have wanted to talk about for a long time.
> > Elementor updates have been very powerful and big for about 3 months. This requires the participation of many, many people to participate in the beta version. Right now, there are maybe only 5 of us who are constantly contributing to beta versions, and that's really low for Elementor, which has millions of users.
> > **I think the reason for low user participation is that they don't know how to contribute to beta versions.** On the other hand, many users may not have licensed the pro version to try it out. I think it's very, very important to get as many people involved in the beta version. This makes the official update much more stable(Of course, this requires that on the other hand the bug fixing team can cover the possible bugs reported in the beta version. **The number of team members.**).
> > **So I wanted to suggest 3 ways that can definitely increase the number of beta contributors.**
> > 1- Whenever a beta version is released, announce it on Instagram, YouTube, and other Elementor platforms in addition to the current email.
> > 2- Publish (in full) a beta participation guide on YouTube, Instagram, and even this GitHub. Things like installing the beta version, where to report bugs on GitHub and other things needed.
> > 3- I don't know how, but somehow make the pro beta available to users for free. As I said above, there are many people who buy the pro version just for their clients and don't have the pro version for themselves to be able to participate in the pro beta version.
> > I hope it was useful.
> > thank you
> 
> According to Elementor website they have 400 employees and obviously a huge money making business model. There should be no need for us to report even the most simply bugs that you can find after playing around with it for 5min.
> 
> That shows the low ambition/quality awareness of Elementor. If they do not learn from other companies making the mistake "we are the best and nothing can happen to us" they will eventually come to a fall. I hope the managament of Elementor can make a turn NOW and finally focus on the community feedbacks and requests.

Wait until AI-generated static sites hit later this year...

**Comment by WTFRUThinking** [comment]
If this doesn't get changed soon I'm done with Elementor. I don't know why this company continually insists on shoving sticks in its own front bike tire while rolling down the road I, for one, and sick and f***ing tired of it. You clearly don't want our opinions, nor our business. Either that, or whoever is making these idiotic calls is the biggest moron on the face of the planet.

**Comment by mbaierl** [comment]
+1 from my side.. the new dark color scheme is really hard to read... the contrast was MUCH better before 3.12 and I'm switching back to light mode on all my sites now...

Please fix this and use bolder fonts, lighter font colors (#FFF would be my recommendation ;-) ).

Thanks.

**Comment by joehdevwp** [comment]
The new semi-transparent pink-on-black color scheme of the 3.12 UI is absolutely ridiculous. I still use Elementor but I use it in spite of, not because of, these so-called "improvements".

**Comment by Daveden2** [comment]
Just to add, even in the latest beta release 3.13 beta 4, the dropdown options for the dark colour scheme is still unreadable. See the image below.
<img width="226" alt="image" src="https://user-images.githubusercontent.com/105490669/236320840-8ae4f791-374f-4a41-b48b-7d7ca3830422.png">


The solution is to simply apply a dark background colour to the <select> in the editor's CSS when the colour scheme is dark.
.e-input-style, input, select, textarea{
background-color: var(--e-a-dark-bg)
}
<img width="226" alt="image" src="https://user-images.githubusercontent.com/105490669/236321100-635c41b3-4ec6-4d03-9b45-f1577af9a361.png">

**Comment by black-eye** [comment]
> Just to add, even in the latest beta release 3.13 beta 4, the dropdown options for the dark colour scheme is still unreadable. See the image below.
> <img alt="image" width="226" src="https://user-images.githubusercontent.com/105490669/236320840-8ae4f791-374f-4a41-b48b-7d7ca3830422.png">

Yeah, I love this one - this is the problem of the new UI in a nutshell: "Which UI theme would you like to choose?" And then (almost) white text on the white BG. Changes made for better accesibility...

**Comment by Daveden2** [comment]
I think I noticed the issue in the editor, there is a background of none for inputs and I'm not sure about the reasoning behind it:

```
.e-input-style,
input,
select,
textarea {
 color:var(--e-a-color-txt);
 border-radius:var(--e-a-border-radius);
 font-size:12px;
 width:100%;
 /* background:none; */
 box-shadow:none;
 border:var(--e-a-border-bold);
 outline:none
}
```

<img width="507" alt="Screenshot 2023-05-04 230929" src="https://user-images.githubusercontent.com/105490669/236341551-aee1afc0-dda9-4741-b30c-12a1c412a39a.png">

**Comment by dixonge** [comment]
The 2nd image example in the OP is what I first noticed. Literally impossible to see the white icon with an almost white highlight. Simple solution - turn off dark mode. Which kinda defeats the purpose of even *having* a dark mode ...

**Comment by edknewseed** [comment]
As a visually impaired person, it is extremely important that this grey text on white be resolved. I literally have to get my wife to come look for things I am trying to read/do.  It's been 7 weeks since this was added to the github and still no fix.  I hope they push this to the top of the list for the next release.

In regards to beta testers, 5 active testers is not enough, but even still, beta testers are not the same as real world users.   

Anyway, I do appreciate all their hard work including Elementor and those here on GitHub.

**Comment by IMTanuki** [comment]
@edknewseed - this is ridiculous. The whole point of the new color scheme was to make it easier for visually  impaired users.

It's nearly impossible for non-impaired users to use - some field names are unreadable.

Now you're sayinhg that even the target users can't read it.

It seems their product manager handed this important task to a dev team that had no clue.

Given their user base, you would have thought they would provide a theme toggle. And absent a theme toggle, to your point, thjey should have had an expanded beta tester group.

**Comment by edknewseed** [comment]
@IMTanuki :  Elementor does give you the option to select from 'light' and 'dark'.  They should've never touched those two and just simply add new themes instead.

As for beta testers, they should give incentives of some sort. Let's say the top 20 beta testers that are consistently active should be given a credit of some kind to their existing plan.  The point is, is that they need to attract beta testers that are willing to put in the effort to tests their software and you do that by giving them a reason to be pro-active in testing their product.
